client/src/components/BurgerMenu.js

```diff
@@ -38,6 +38,46 @@ function BurgerMenu() {
 export default BurgerMenu;
 
 const Menu = styled.section`
+  border: groove 0.2rem goldenrod;
```

Comment:
maybe you could consider sorting your styles in any kind of order, for example alphabetically? :D